src/javaRosa.js

```diff
@@ -1993,7 +1993,9 @@ define([
             };
             databind.constraintMediaIText = function (mugOptions) {
                 return mugOptions.isSpecialGroup ? undefined : {
-                    visibility: 'constraintMsgItext',
+                    visibility: function(mug) {
+                        return mug.__className !== "DataBindOnly" && mug.isVisible("constraintMsgItext");
```

Comment:
I think `mug.isVisible("constraintAttr")` will do the same thing and is more maintainable